templates/server/src/main/resources/archetype-resources/core/src/main/resources/application.properties

```diff
@@ -59,7 +59,7 @@ spring.batch.job.enabled=false
 
 # Flyway for Database Setup and Migrations
 #if ($dbType == 'mariadb')
-flyway.locations=classpath:db/migration,classpath:db/type/mysql
+#spring.flyway.locations=classpath:db/migration,classpath:db/type/mysql
```

Comment:
why was that commented out? Mistake or on purpose?

Reply:
Verified. Indeed a small mistake. Big change and aspect not covered by tests. I will fix this with a PR.